src/tkn.ts

```diff
@@ -195,7 +195,7 @@ export class Command {
   }
   @verbose
   static listPipelineRuns(name: string): CliCommand {
-    return newTknCommand('pipelinerun', 'list', name, '-o', 'json');
+    return newK8sCommand('get', 'pipelinerun', '-l', `tekton.dev/pipeline=${name}`, '-o', 'json');
```

Comment:
can we show the running pipelineRun and taskRun at the top? I think when we were using `tkn` command that time we were not facing that problem.

<img width="446" alt="Screenshot 2020-04-17 at 2 31 28 PM" src="https://user-images.githubusercontent.com/9924513/79552148-44831300-80b8-11ea-9145-1567209549e4.png">